### PR TITLE
fix rounding error in effective_parameters tests

### PR DESCRIPTION
**Pull request by koniecsveta**

### CODE REVIEW

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 1]
```diff
@@ -4357,3 +4357,9 @@ def assertCoefEqual(regCoeff, coeff, coeffClassSet, tol=1e-6):
         diff = abs(val1-val2)
         print("val1: {0}, val2: {1}, tol: {2}".format(val1, val2, tol))
         assert diff < tol, "diff {0} exceeds tolerance {1}.".format(diff, tol)
+
+def almost_equal(x, y, tol=1e-6):
```
Comment (Pscheidl) [inline h2o-py/tests/pyunit_utils/utilsPY.py:4361]: Just name `is_equal` please.

**Review by Pscheidl — APPROVED** [review]

File: h2o-py/tests/pyunit_utils/utilsPY.py (hunk 1 above)
Comment (Pscheidl) [reply h2o-py/tests/pyunit_utils/utilsPY.py:4361]: Equals is probably better.

Comment (Pscheidl) [reply h2o-py/tests/pyunit_utils/utilsPY.py:4361]: Also, limit the arg type if possible in python.

Comment (koniecsveta) [reply h2o-py/tests/pyunit_utils/utilsPY.py]: fixed

**Review by Pscheidl — APPROVED** [review]

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 2]
```diff
@@ -4357,3 +4357,12 @@ def assertCoefEqual(regCoeff, coeff, coeffClassSet, tol=1e-6):
         diff = abs(val1-val2)
         print("val1: {0}, val2: {1}, tol: {2}".format(val1, val2, tol))
         assert diff < tol, "diff {0} exceeds tolerance {1}.".format(diff, tol)
+
+def equals(x, y, tol=1e-6):
+    if isinstance(x, (int, float)) & isinstance(y, (int, float)):
+        if abs(x-y) > tol:
+            assert False, "Value {0} and value {1} do not agree.".format(x, y)
```
Comment (satai) [inline h2o-py/tests/pyunit_utils/utilsPY.py:4364]: `agree` seems not to be the right word here?

Comment (satai) [reply h2o-py/tests/pyunit_utils/utilsPY.py:4364]: OH, too late, never mind.

**Comment by satai** [comment]
btw you can `import numpy.testing` (already available, no need to make a new image to get it) that has `assert_almost_equal` that works for arrays too.

**Pull request merged**